frontend/src/components/TakeQuiz.jsx

```diff
@@ -24,8 +24,116 @@ const questions = [
     question: "I feel goofy most days",
     answers: [5, 4, 3, 2, 1],
   },
+  {
```

Comment:
So right now, the questions, answers, and moods are all strongly coupled. What you could do is create an enum for moods and have each question have a mood field. That way, you could iterate over the answers and know which mood to adjust. This would make your code much more modular, and make it much easier to have multiple questions for a single mood, or potentially even allow users to create questions/quizes.